sample_data/docker-compose-dl-streamer-example-all.yml

```diff
@@ -0,0 +1,506 @@
+# SPDX-FileCopyrightText: (C) 2025 Intel Corporation
```

Comment:
How about we take alternative approach and use [compose profiles](https://docs.docker.com/compose/how-tos/profiles/) and keep one compose file for both core and experimental services. Seems simpler to maintain.

Services without profile start by default, so we can keep core without any modifications and add e.g. `experimental` profile to mapping service, so that it only starts with `docker compose --profile experimental up`.